osm2gtfs/creators/ci_abidjan/stops_creator_ci_abidjan.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
```

Comment:
I suggest we remove this file and use the default creator instead

Reply:
C'est à dire que l'on arrête d'essayer de faire les stop areas. Ca me va.

Reply:
DONE

Reply:
En cours, ça crée plein d'erreurs, qu'il faut investiguer. Je fais ça après le reste.